src/main/java/com/microsoft/jenkins/keyvault/SecretCertificateCredentials.java

```diff
@@ -85,7 +85,7 @@ public KeyStore getKeyStore() {
         }
 
         try {
-            final byte[] content = Base64.decodeBase64(secret.getValue());
+            final byte[] content = Base64.getMimeDecoder().decode(secret.getValue());
```

Comment:
Why mine decoder?

Reply:
Otherwise the test failed [ci.jenkins.io](https://ci.jenkins.io/job/Plugins/job/azure-credentials-plugin/view/change-requests/job/PR-217/1/testReport/com.microsoft.jenkins.keyvault/SecretCertificateCredentialsTest/getKeyStore/)